Examples/microcontroller/STM32F302R6T6TR.json

```diff
@@ -0,0 +1,11407 @@
+            "relativeValueOperator": "add",
+            "conditions": [
+              "CMOS port",
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "minValue": 2.4,
+            "conditions": [
+              "TTL port",
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": -1.3,
+            "relativeValueOperator": "add",
+            "conditions": [
+              "IIO = +20 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": -0.4,
+            "relativeValueOperator": "add",
+            "conditions": [
+              "IO = +6 mA",
+              "2 V < VDD < 2.7 V"
+            ]
+          }
+        ]
+      },
+      "absVmax": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "maxValue": 4
+          }
+        ]
+      },
+      "absVmin": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VSS",
+            "relativeValueModifier": -0.3,
+            "relativeValueOperator": "add"
+          }
+        ]
+      },
+      "vmax": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDDA",
+            "relativeValueModifier": 0.3,
+            "relativeValueOperator": "add"
+          }
+        ]
+      },
+      "imax": {
+        "values": [
+          {
+            "siUnit": "milliamp",
+            "maxValue": 25
+          }
+        ]
+      },
+      "inputLeakage": {
+        "values": [
+          {
+            "siUnit": "microamp",
+            "maxValue": 0.1,
+            "conditions": [
+              "digital mode",
+              "VSS ≤ VIN ≤ VDD"
+            ]
+          },
+          {
+            "siUnit": "microamp",
+            "maxValue": 1,
+            "conditions": [
+              "digital mode",
+              "VDD ≤ VIN ≤ VDDA"
+            ]
+          },
+          {
+            "siUnit": "microamp",
+            "maxValue": 0.2,
+            "conditions": [
+              "analog mode",
+              "VSS ≤ VIN ≤ VDDA"
+            ]
+          }
+        ]
+      },
+      "esd": true
+    },
+    {
+      "terminalIdentifier": "37",
+      "name": "PC6",
+      "numberOfSupportedFunctions": 4,
+      "functionProperties": [
+        {
+          "perFunctionName": "PC6",
+          "direction": "bidir"
+        },
+        {
+          "perFunctionName": "EVENTOUT"
+        },
+        {
+          "perFunctionName": "I2S2_MCK",
+          "interfaceType": "i2c",
+          "electricalConfiguration": "open-drain"
```

Comment:
interface - i2s. In general, I think the code is missing the i2s option

Reply:
yeah , I just used SPI, but the problem was that my regex was bad. that has been fixed.